.github/workflows/deploy.yml

```diff
@@ -0,0 +1,61 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  # Could be disabled if we don't want it to automatically deployed when merging a PR
+  push:
+    branches: ['main']
```

Comment:
Do we want to deploy of `main`, or off a dedicated `github-pages` branch?
I’m ambivalent, both have their pros and cons.

Reply:
Reason why I left it to be deploy on push to main is because it is lower maintenance compared to deploying off a dedicated `github-pages` branch

Reply:
Fair. `main` it is, then!